frontend-ssr/components/exercise-submissions/SubmissionSkeleton.module.scss

```diff
@@ -0,0 +1,42 @@
+@import '../../styles/variables';
```

Comment:
All this code is duplicated from the other SCSS file. This should have "ringed" a warning bell and make you question if this is the right approach. Let's merge the two style files and remove the duplicated code.